.github/workflows/releaseWithCoreBundle.yml

```diff
@@ -16,8 +17,8 @@ on:
 
 jobs:
   call-release-workflow:
-    if: ${{ inputs.branch || (github.event_name == 'workflow_run' && github.event.workflow_run.conclusion == 'success')}}
+    if: ${{ inputs.branch }}
     uses: forcedotcom/bundle-publish-scripts/.github/workflows/releaseWithCoreBundle.yml@main
     secrets: inherit
     with:
-      branch: ${{ inputs.branch || 'main'}}
+      branch: ${{ inputs.branch }}
```

Comment:
fyi @iowillhoit 
this value isn't used in a `run` block and seems safe, see:

https://github.com/forcedotcom/bundle-publish-scripts/blob/3e83ccd7faaa84316facb1c35ed0b2ebba073f06/.github/workflows/releaseWithCoreBundle.yml#L17